### DISCUSSION TITLE
How to print multiple jobs using "collate printing"

### DISCUSSION THREAD

**Discussion by ZJuntao** [sumatrapdfreader/sumatrapdf#3915] [Q&A] [answered]
i find that the options of sumatrapdf tool cannot print multiple jobs using "collate printing", this option should be an universal feature for printing doc, I hope you can help me to answer how to use this function, thank you!

**Comment by GitHubRulesOK** [comment, marked as answer]
Microsoft probably removed that with newer dialog, are you on Windows 11 ?, but SumatraPDF can only offer whatever the print driver generates as default options, so the setting may need change in that printers default settings.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/98f1acda-46d5-49d1-a950-87bff2b189be)

**Reply by ZJuntao** [reply]
Okay，I got it, thank you!!!
